ReactNativeClient/lib/components/screens/note.js

```diff
@@ -121,6 +121,19 @@ class NoteScreenComponent extends BaseScreenComponent {
 				return true;
 			}
 
+			if (Platform.OS === 'android' && this.state.fromShare) {
```

Comment:
Can `state.fromShare` be set for iOS? If not, we can remove the platform check.

Reply:
Will do. Since I have no way of testing iOS I was maybe too defensive about trying not to break anything for iOS

Reply:
Done now